src/OrchardCore/OrchardCore/Shell/Configuration/ShellConfigurationSources.cs

```diff
@@ -70,5 +73,28 @@ public async Task SaveAsync(string tenant, IDictionary<string, string> data)
                 }
             }
         }
+
+        public Task RemoveAsync(string tenant)
+        {
+            var tenantFolder = Path.Combine(_container, tenant);
+            var appsettings = Path.Combine(tenantFolder, "appsettings.json");
+
+            if (File.Exists(appsettings))
+            {
+                File.Delete(appsettings);
```

Comment:
Why we have this, while it's already there in line 87?

Reply:
Oops ;) I will delete it, thanks